gemet/thesaurus/management/commands/check_spreadsheet.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+from gemet.thesaurus.models import Property
+
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+import re
+
+
+LABELS = {
+    'RT (GEMET)': True,
+    'RT (new)': False,
+    'BT (GEMET)': True,
+    'BT (new)': False,
+    'NT (GEMET)': True,
+    'NT (new)': False,
+}
+
+
+def check_term_database_existence(term):
+    return Property.objects.filter(value=term).exists()
+
+
+def split_text_into_terms(raw_text):
+    pattern = re.compile("[^a-zA-Z\d \-\\)\\(:]")
+    term_list = pattern.split(raw_text)
+    term_list = map(lambda term: term.strip(' \t\n;,.:').lower(), term_list)
```

Comment:
You have no reason to strip all of these characters: ` \t\n;,.:` because only ` ` (space) and `:` (colon) can be present in the string. All other characters are included in the pattern used to split the string, so they will be removed from the resulting substrings.